lkml/lexer.py

```diff
@@ -166,10 +148,15 @@ def scan_literal(self) -> tokens.LiteralToken:
         return tokens.LiteralToken(chars, self.line_number)
 
     def scan_quoted_literal(self) -> tokens.QuotedLiteralToken:
-        """Tokenize quoted literals.
+        """Returns a token from a quoted literal string.
+
+        The initial double quote character is consumed in the scan method, so this
+        method only scans for the trailing quote to indicate the end of the token.
 
-        Returns:
-            tokens.QuotedLiteralToken
+        Example:
+            >>> lexer = Lexer('Label"')
```

Comment:
typo

Reply:
Sorry, what's the typo? If you're referring to the omitted leading double quote, that has to be missing because it's dropped by the lexer in the `scan` method before this one gets called.

Reply:
You're right--my bad!